package/msys/PKGBUILD

```diff
@@ -0,0 +1,67 @@
+# Author: williamjcm <w.jcm59@gmail.com>
+# Contributor: mosra <mosra@centrum.cz>
+# Based on the packages/archlinux/PKGBUILD file by mosra <mosra@centrum.cz> and the MSYS2/MinGW PKGBUILD templates
+
+_realname=magnum
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+pkgver=dev
+pkgrel=1
+pkgdesc="C++11/C++14 graphics middleware for games and data visualization"
+arch=('any')
+url="https://magnum.graphics/"
+license=('MIT')
+# The corrade-git dependency corresponds to the package generated by Corrade's packages/msys/PKGBUILD file.
+depends=("${MINGW_PACKAGE_PREFIX}-corrade-git"
```

Comment:
Isn't the corrade package called just `${MINGW_PACKAGE_PREFIX}-corrade`? Same for the `-git` suffixes in `pkgbase` and `pkgname`.

Since this builds current working directory (instead of pulling a version from git), I wouldn't put a `-git` suffix on it.

Reply:
I think the `-git` suffix is also in the equivalent Corrade PKGBUILD, so it should also be removed there.

Reply:
Oh, huh, right. Not sure where I was looking then.

Will push that right away together with the `git` dependency addition.